setup.py

```diff
@@ -40,6 +40,5 @@
         "matplotlib",
         "pymatgen>=2017.12.30",
     ],
-    },
```

Comment:
Misplaced close bracket. This code wouldn't run!

Reply:
ouch lol ty for checking